feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -313,15 +315,16 @@ func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, ateTop gosnappi.
 	ate.OTG().StopTraffic(t)
 	otgutils.LogFlowMetrics(t, ate.OTG(), ateTop)
 	otgutils.LogPortMetrics(t, ate.OTG(), ateTop)
-	flowPath := gnmi.OTG().Flow(flow)
-	got := gnmi.Get(t, ate.OTG(), flowPath.LossPct().State())
+	recvMetric := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flow).State())
+	lostPackets := recvMetric.GetCounters().GetOutPkts() - recvMetric.GetCounters().GetInPkts()
+	got := float32(lostPackets * 100 / recvMetric.GetCounters().GetOutPkts())
```

Comment:
We need to cast `lostPackets` and `recvMetric.GetCounters().GetOutPkts()` individually, otherwise the calculation inside the parentheses is still going to result in an integer value.)

Reply:
Code changed